i18n/translations/en.json

```diff
@@ -413,16 +413,15 @@
 
 	"license": "License",
 	"license.activate": "Activate it now",
+	"license.activate.label": "Please activate your site",
+	"license.activate.domain": "Your license will be activated for <strong>{host}</strong>.",
+	"license.activate.local": "You are about to use your Kirby license to activate your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please activate it there instead. If {host} is the domain you want to register with your license, please continue.",
```

Comment:
```suggestion
	"license.activate.local": "You are about to activate your Kirby license for your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please activate it there instead. If {host} is the domain you want to license Kirby to, please continue.",
```

Reply:
"the domain you want to license Kirby to" sounds a bit odd.
What about "the domain you want to use your Kirby license for"?

Reply:
Omitted Kirby for brevity (additional line break), otherwise 👍